packages/nimble-components/src/anchor-tabs/specs/README.md

```diff
@@ -0,0 +1,204 @@
+- Supports keyboard navigation within group of tabs without automatically activating each focused tab
+- Disable individual tabs
+
+### Risks and Challenges
+
+None
+
+### Prior Art/Examples
+
+In SLE:
+
+![SLE](sle-systems.png)
+
+In GitHub:
+
+![GitHub](github.png)
+
+---
+
+## Design
+
+Anchor tabs have the same appearance as a standard tabs control, i.e. a sequence of operable regions with labels, each of which corresponds to some target content to display. In the case of anchor tabs, they do not switch between pre-configured tab panels, but rather navigate to some URL. The tabs can be configured to load the URL in a specific target, like an `iframe`.
+
+The design consists of an "Anchor tabs" container component which contains one or more "Anchor tab" components. This mirrors the existing design of the `nimble-tabs` and `nimble-tab` components, however there is no analog for the `nimble-tab-panel` component. Instead, the content loaded from the configured URL(s) loads into the containing page, an `iframe`, a new window, etc.
+
+We cannot extend FAST's tabs control because it has baked into it the idea that tabs activate tab panels. Instead we will create new components from scratch, but we can likely borrow logic from FAST e.g. for handling keyboard navigation.
+
+### API
+
+#### Anchor Tabs
+
+- *Component Name* - `nimble-anchor-tabs`
+- *Props/Attrs*
+    -   `activeid` - id of active tab. Defaults to the first tab.
+- *Methods* - None
+- *Events* - None
+- *CSS Classes and CSS Custom Properties that affect the component* - None
+- *Slots*
+    -   `start` - content placed to the left of tab list
+    -   `anchortabs` - slot for anchor tab elements
+    -   `end` - content placed to the right of tab list
+
+#### Anchor Tab
+
+- *Component Name* - `nimble-anchor-tab`
+- *Props/Attrs*
+    -   `download` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `href` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes). Remove to disable the tab.
+    -   `hreflang` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `ping` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `referrerpolicy` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `rel` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `target` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `type` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+- *Methods* - None
+- *Events* - None
+- *CSS Classes and CSS Custom Properties that affect the component*
+    -   `active` - class indicating that the tab is active within the containing tabs group
```

Comment:
I wasn't intending for them to be part of the public API. I just figured I would need them as part of the implementation and wanted to reflect that here. I guess this should just be the public API, though, so I'm removing them.

Reply:
I think we need to be careful about setting classes on elements that are created & owned by a client application because we don't want to conflict with any styling that a client is doing. We also don't want clients to style any elements based on these classes we're adding (for all the reasons why @rajsite changed our other APIs not too use CSS classes as part of the API).

Is there a different approach we can take rather than using CSS classes? It looks like the regular tabs style things based off of `aria-selected='true'`.

Reply:
As Milan suggested in a side conversation, I think we may be able to actually focus individual tabs rather than keeping the focus on the tabs control itself. That would allow us an easy way to style the focused tab appropriately, and we can use the value of `aria-selected` to style the selected tab, just as you said.